full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -188,6 +191,90 @@ describe("CloudSqlite", () => {
     container.disconnect({detach: true});
   });
 
+  it("Should upload a single user's changes successfully when a write lock is expired", async () => {
+    const testContainer1 = testContainers[0];
+    const testCache = azSqlite.makeCaches(["testCache"])[0];
+    testContainer1.connect(testCache);
+    testContainer1.acquireWriteLock(user1);
+    await testContainer1.copyDatabase("testBim", "testBimCopy");
+    const db = await BriefcaseDb.open({ fileName: "testBimCopy", container: testContainer1 });
+    db.saveFileProperty({ name: "logMask", namespace: "logMaskTest", id: 1, subId: 1 }, "this is a test");
+    db.close();
+    // set the expires time to five mins earlier, which avoids waiting for 5 mins here.
+    await azSqlite.subtractFromCurrentWriteLockExpiryTime(testContainer1, "bcv_kv.bcv", 5);
```

Comment:
Is the write lock actually expired here? isn't testContainers[0] not passing a lockExpireSeconds which means we default to 1hr for the write lock?

Reply:
We should assert that our assumption the write lock is expired is true using that new function you added

Reply:
Resolved in [commit b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8](https://github.com/iTwin/itwinjs-core/pull/7000/commits/b6de82106a3f57bb9db95b4dbd3dcfd47076e2c8)